assets/wizards/connections/views/main/google.js

```diff
@@ -60,6 +60,25 @@ const GoogleOAuth = ( { setError, canBeConnected } ) => {
 	const [ inFlight, setInFlight ] = useState( false );
 	const handleError = res => setError( res.message || __( 'Something went wrong.', 'newspack' ) );
 
+	useEffect( () => {
+		if ( isConnected && ! userBasicInfo.has_refresh_token ) {
```

Comment:
Whenever I make changes to the `_newspack_google_oauth` option, when I `console.log ( userBasicInfo )` I see `undefined`, so `isConnected` is always false due to the lack of an `email` property.